django/conf/project_template/manage.py-tpl

```diff
@@ -1,3 +1,7 @@
+"""
```

Comment:
Please use a single line docstring and don't use a curly quote in "Django's":
```
"""..."""
```
Also, I think the docstring should be [after the shebang](https://stackoverflow.com/a/2397564/5112), without a blank line before or after.

Reply:
Fixed.